packages/provider/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@trustwallet/provider",
+  "version": "0.0.12",
+  "description": "Trust provider wrapper",
+  "author": "TrsutWallet",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/trustwallet/web-core/issues"
+  },
+  "homepage": "https://github.com/trustwallet/web-core#readme",
+  "keywords": [],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/trustwallet/web-core.git"
+  },
+  "scripts": {
+    "clean": "rm -rf lib",
+    "build": "webpack",
+    "test": "mocha --require ./babel-polyfill.js test/**/*.spec.ts"
+  },
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "files": [
+    "lib",
+    "src"
+  ],
+  "devDependencies": {
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/node": "^7.0.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-typescript": "^7.1.0",
+    "@babel/register": "^7.0.0",
+    "@types/chai": "^4.1.6",
```

Comment:
I think using jest is better than mocha/chai. Desktop already uses jest for everything. I think it'd be good to use the testing env from that.

Reply:
Good catch. I'll remove it